test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -43,6 +43,27 @@ TEST(OutlierDetectorImplFactoryTest, Detector) {
             DetectorImplFactory::createForCluster(cluster, *loader, dispatcher, runtime, nullptr));
 }
 
+TEST(OutlierDetectorImplFactoryTest, DetectorConfig) {
```

Comment:
this is not really a useful test. Can you test the config object directly, and make sure the loaded values are what you expect (or do that additionally)